setup.py

```diff
@@ -58,7 +58,7 @@ def run_tests(self):
     # Versions should comply with PEP440. For a discussion on
     # single-sourcing the version across setup.py and the project code,
     # see http://packaging.python.org/en/latest/tutorial.html#version
-    version='4.4.0',
+    version='4.5.0',
```

Comment:
I do not think you should touch this as version is controlled by bumpversion, there is a `.bumpversion.cfg` in the repository. However this is unclear since there are no `DEPLOYMENT.md`-instructions. Could you add a file more or less like this: https://github.com/Clinical-Genomics/housekeeper/blob/master/DEPLOYMENT.md just replacing housekeeper. And then after merging running bumbversion command together with `git push` and `git push --tag`? 

Tell if you need any assistance!

Reply:
Fixed I think!